docs/measuring_performance/measuring_performance_rg1.md

```diff
@@ -0,0 +1,96 @@
+---
+---
+
+        <h3>
+          <a id="user-content-note-this-document-is-related-to-radargun-1x" class="anchor" href="#note-this-document-is-related-to-radargun-1x" aria-hidden="true"></a>Note: this document is related to RadarGun 1.x</h3>
```

Comment:
Ok, I was not sure whether to keep it or lose it.
The versions - bigger topic, will talk to you in IRC